apps/studio/components/interfaces/JwtSecrets/jwt-secret-keys-table/create-key-dialog.tsx

```diff
@@ -24,8 +23,6 @@ import {
   Textarea,
 } from 'ui'
 
-dayjs.extend(relativeTime)
```

Comment:
is this unnecessary? just want to know so that if I see it anywhere else I can remove it

Reply:
yup the dayjs extensions are all done on the _app level, so no need to define these in the individual components